app/kuma-dp/cmd/run.go

```diff
@@ -75,7 +76,7 @@ func newRunCmd(opts kuma_cmd.RunCmdOpts, rootCtx *RootContext) *cobra.Command {
 
 			var tpCfg *tproxy_dp.DataplaneConfig
 			if len(tpCfgValues) > 0 || tpEnabled {
-				if runtime.GOOS != "linux" {
+				if !testing.Testing() && runtime.GOOS != "linux" {
```

Comment:
why do we need this?

Reply:
I'll add tests for `kuma-dp run` with the tproxy flags, but that probably belongs in the other PR, not this one.